codex-ui/dev/pages/layout/Navbar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="navbar-wrapper">
+    <Navbar>
+      Left
+      <template #right>
+        Right
+      </template>
+    </Navbar>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Navbar } from '../../../src/vue';
+</script>
+
+<style scoped>
+.navbar-wrapper {
+    min-height: 400px;
+    position: relative;
+    background-color: var(--base--bg-secondary);
+    border-radius: var(--radius-m);
+    border: 1px solid var(--base--border);
+    overflow-x: clip;
+
+    .navbar {
+        z-index: 1;
+        top: var(--layout-navbar-height);
+        border-radius: var(--radius-m) var(--radius-m) 0 0;
+        padding: 0 var(--spacing-ms);
```

Comment:
I think, we should include horizontal paddings to the Navbar. And make it bigger. Right now app-navbar also has too small paddings:

<img width="372" alt="image" src="https://github.com/user-attachments/assets/9a7db6a5-369b-4879-a224-bb38a95c1a42" />